### PR TITLE
refactored flag variable default values into consts

### PR DESCRIPTION
**Pull request by souvikmaji**
refers to  #64

### CODE REVIEW

File: cmd/proc.go
Comment (MadhavJivrajani) [inline cmd/proc.go:117]:
```suggestion
		"specify pid of process",
	)
```

**Review by MadhavJivrajani — CHANGES_REQUESTED** [review]
- The check for valid PID can be done using the newly defined consts
```go
if pid != defaultProcPid {
    ...
}
```

- Feel free to remove the extra comments in the `init()` function.

**Comment by souvikmaji** [comment]
What validation should i add for pids?

**Comment by MadhavJivrajani** [comment]
Validation is already taken care of, since you've added the const `defaultProcPid`, you could use that in the check on [line 64](https://github.com/souvikmaji/grofer/blob/94f1682908b6d9571f44fe05dc307ee4ff94e818/cmd/proc.go#L64) so that if a default value should be changed, it only needs to be changed in the defined consts and not anywhere else.

**Review by MadhavJivrajani — APPROVED** [review]
Looks good! Thanks for your contribution! I'll wait for another review from either @Gituser143 or @metonymic-smokey before merging.

**Review by metonymic-smokey — APPROVED** [review]

**Comment by MadhavJivrajani** [comment]
@souvikmaji a PR for changing PID types has already been made and ideally should not be a part of this PR. Please revert those changes for now.

**Comment by souvikmaji** [comment]
Reverted

**Review by MadhavJivrajani — APPROVED** [review]

**Pull request merged**
